.circleci/config.yml

```
@@ -0,0 +1,122 @@
+version: 2
+
+defaults: &defaults
+  working_directory: /tmp/project
+  environment:
+    APM_TEST_PACKAGES: "language-lua"
+  docker:
+    - image: arcanemagus/atom-docker-ci:stable
+  steps:
+    # Restore project state
+    - attach_workspace:
+        at: /tmp
+    - run:
+        name: Install luac and luajit
+        # Note, using ruby-full instead of ruby for the headers to compile rubocop
+        command: |
+          sudo apt-get update && \
+          sudo apt-get install --assume-yes --quiet --no-install-suggests \
+            --no-install-recommends lua5.1 luajit
```

Looks like there is a `lua5.3` package, might as well use the current one.
```suggestion
            --no-install-recommends lua5.3 luajit
```

---

Seems like lua 5.3 is not available on trusty repo.